### PR TITLE
Default to more iterations in Room.is_inside to avoid failure of the method

### PR DESCRIPTION
**Pull request by grantgasser**
Change made because of Issue #150

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -308,3 +308,12 @@ Changed
 .. _0.1.16: https://github.com/LCAV/pyroomacoustics/compare/v0.1.15...v0.1.16
 .. _0.1.15: https://github.com/LCAV/pyroomacoustics/compare/v0.1.14...v0.1.15
 .. _0.1.14: https://github.com/LCAV/pyroomacoustics/compare/v0.1.13...v0.1.14
+
+`0.1.14`_ - 2018-02-20
```
[the remark this answers is not in the document]
Comment (grantgasser) [reply CHANGELOG.rst]: Please see [098ffb3](https://github.com/LCAV/pyroomacoustics/pull/151/commits/098ffb3701b86e386f8d0317d024a434f9efe37e) for changes

[the remark this answers is not in the document]
Comment (fakufaku) [reply CHANGELOG.rst]: Thanks a lot!

[the remark this answers is not in the document]
Comment (grantgasser) [reply CHANGELOG.rst]: @fakufaku Is this ready to merge?

[the remark this answers is not in the document]
Comment (fakufaku) [reply CHANGELOG.rst]: Yes, sorry to keep you waiting!

**Comment by fakufaku** [comment]
Thank you for your contribution!

**Pull request merged**